cli/internal/fs/turbo_json.go

```diff
@@ -179,6 +183,10 @@ func ReadTurboConfig(rootPath turbopath.AbsoluteSystemPath, rootPackageJSON *Pac
 			rootPackageJSON.LegacyTurboConfig = nil
 		}
 
+		if hasPackageJSONTurboVersion && turboJSON.TurboVersion != "" {
+			return nil, fmt.Errorf("Found versions for turbo in both turbo.json and root package.json. Please delete the package.json dependency")
+		}
```

Comment:
What happens if there's a `turbo` key in `package.json`? test case for that might also be good. Eg. if package.json is:

```json
{
	"turbo": {
		"turboVersion": "1.7.0"
	}
}
```